app/assets/javascripts/Measure.js

```diff
@@ -53,14 +49,12 @@ const MeasureSchema = mongoose.Schema(
     populations_cql_map: Mixed,
     observations: [Mixed],
 
-    value_sets: [{ type: ObjectId, ref: 'ValueSet' }],
-
-    complexity: Mixed, // Bonnie-specific?
-
     // Relations to other model classes
-    user: { type: ObjectId, ref: 'User', index: true }, // Bonnie-specific
-    bundle: { type: ObjectId, ref: 'Bundle' }, // Cypress-specific, but used in Bonnie-bundler as well
-    package: { type: ObjectId, ref: 'MeasurePackage' }, // Bonnie specific
+    bundle: { type: ObjectId, ref: 'Bundle' }, // Cypress-specific, until we migrate the Bundle into cqm-models
+    package: { type: ObjectId, ref: 'MeasurePackage' }, // Bonnie-specific
+    patients: [{ type: ObjectId, ref: 'Patient', index: true }],
+
+    value_sets: [{ type: ObjectId, ref: 'ValueSet' }],
```

Comment:
does this align w the new model for handling value sets?

Reply:
This does, yes. @hossenlopp's document has Measure objects having many `ValueSet` objects, and this is how you do that in Mongoose (a non-embedded many relation).